### ISSUE TITLE
Need adding empty line before fun

### ISSUE THREAD

**Issue by afanmatvey** [shyiko/ktlint#87]
I would like to add this check

### Case 1: 

Fails:
```
interface Rule {
    fun check()
    fun test(enabled: Boolean)
}
``` 

Success:
```
interface Rule {

    fun check()

    fun test(enabled: Boolean)
}
```

### Case 2: 

Fails:
```
    fun method1() {
        // ...
    }
    fun method2(p: Boolean) {
        // ..
    }
``` 

Success:
```
    fun method1() {
        // ...
    }

    fun method2(p: Boolean) {
        // ..
    }
```

**Comment by vanniktech** [comment]
Correct me if I'm wrong but with 0.14.0, you should get this behavior now thanks to https://github.com/shyiko/ktlint/pull/72/

The only thing you'll still get flagged is

```kotlin
interface Rule {

    fun check()

    fun test(enabled: Boolean)
}
```

since there's no need for that newline between the interface and method declarations. It's unnecessary.
